src/components/Rockets.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRocket } from '../redux/rockets/rocketSlice';
 
-const Rockets = () => (
-  <div>Rockets</div>
-);
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const { rockets } = useSelector((state) => state.rocket);
+  useEffect(() => {
+    dispatch(fetchRocket());
+  });
```

Comment:
Kindly add the dependency array to useEffect, otherwise the website makes infinite requests and crashes the browser.

Reply:
yea you got me Thank you